tests/lib/external/snapd-testing-tools/tests/spread-filter/config.yaml

```diff
@@ -0,0 +1,33 @@
+rules:
+  rule_tests:
+    from: 
```

Comment:
I am wondering if "from" and "to" is the optimal choice here? Is is possible to choose more descriptive keywords?

Reply:
source and target perhaps?

Reply:
perhaps source and target?

Reply:
I used from and to because the rule converts from a set of changes to a set of tests